responsibleai_vision/tests/test_image_utils.py

```diff
@@ -86,3 +87,12 @@ def test_retry_sessions_retries_on_conn_failure(self, request_mock):
         session = image_reader_get_retry_session(url)
 
         assert session.get(url).status_code == 200
+
+    def test_get_all_exif_feature_names(self):
+        image_dataset = load_fridge_object_detection_dataset().head(2)
+        exif_feature_names = get_all_exif_feature_names(image_dataset)
+        assert len(exif_feature_names) == 11
```

Comment:
You could change this line to compare this with the length of the set in lines 95-98.

Reply:
Length of the set or the list would be the same. Just casted the elements to a set for the assertion that the objects have the same elements (coz the order of `exif_feature_names` can change).